pkg/common/config/config.go

```diff
@@ -139,12 +143,12 @@ func FromEnv(ctx context.Context, cfg *Config) error {
 		return fmt.Errorf("config object cannot be nil")
 	}
 	log := logger.GetLogger(ctx)
-	//Init
+	// Init.
 	if cfg.VirtualCenter == nil {
 		cfg.VirtualCenter = make(map[string]*VirtualCenterConfig)
 	}
 
-	//Globals
+	// Globals.
```

Comment:
Do we need to follow the period rule for single words as well? It isn't exactly a sentence in english language

Reply:
+1

Reply:
Yes. We do, when we write vpxd and cns code, to follow a simple rule.

But, it is better for you to write a full sentence. The fair question would be whether "Globals" is a good comment? Do you really need it?